modules/ROOT/partials/configuration/newdocument_content.adoc

```diff
@@ -0,0 +1,21 @@
+[[newdocument_content]]
+== `+newdocument_content+`
+
+This option enables you to set the content that the editor will contain if you use the New document menu item. This can
+be used together with xref:content-behavior-options.adoc#editable_root[editable_root] to make sure there is always a editable field within the non-editable root.
+
+*Type:* `+String+`
+
+*Default value:* `+''+`
+
+=== Example: using `+newdocument_content+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  newdocument_content: '<p contenteditable="true">Editable content</p>' 
+});
```

Comment:
```suggestion
This option sets the content a new editor contains when the xref:available-menu-items.adoc#the-core-menu-items[File -> New document] menu item is invoked.

*Type:* `+String+`

*Default value:* `+''+`

=== Example: using `+newdocument_content+`

This example shows `+newdocument_content+` being used with the xref:content-behavior-options.adoc#editable_root[`+editable_root+`] option to set an editable element within a non-editable root.

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  editable_root: false,
  newdocument_content: '<div contenteditable="true">Editable content</div>' 
});
```
1. Copy-edits and edits.
2. Incorporates @MitchCTiny’s structural edit above.
3. Also adds an xref as per @MitchCTiny’s editorial note.
4. Takes a liberty with the example and changes the tag set in place by`newdocument_content` tag `<p>` to `<div>`.
	* Thinking like the sys-admin I once was for a moment, I’d put my _end-users can edit this part of the form_ chunks inside `<div></div>` tags rather than <p></p> tags.

Reply:
All makes sense thanks for these changed updated the PR.